### PR TITLE
i18n (app.yml): Add Spanish localization (es)

### PR DESCRIPTION
**Pull request by Rikiub**
## What does this PR do
Add Spanish localization (es).

## Standards checklist

- [x] The PR title is descriptive.
- [x] I have read `CONTRIBUTING.md`
- [x] *Optional:* I have tested the code myself
 
## For new steps

- [ ] *Optional:* Topgrade skips this step where needed
- [ ] *Optional:* The `--dry-run` option works with this step
- [ ] *Optional:* The `--yes` option works with this step if it is supported by the underlying command

If you developed a feature or a bug fix for someone else and you do not have the
means to test it, please tag this person here.

### CODE REVIEW

File: locales/app.yml [hunk 1]
```diff
@@ -73,392 +95,518 @@ _version: 2
...
   en: "Topgrade finished with errors"
+  es: "Topgrade finalizado con errores"
   zh_TW: "Topgrade 執行部分成功"
 "Topgrade finished successfully":
   en: "Topgrade finished successfully"
+  es: "Topgrade finalizó exitosamente"
   zh_TW: "Topgrade 執行成功"
 "Topgrade {version_str} Breaking Changes":
   en: "Topgrade %{version_str} Breaking Changes"
+  es: "Topgrade {version_str} Cambios Importantes"
```
Comment (SteveLauC) [inline locales/app.yml:199]:
```suggestion
  es: "Topgrade %{version_str} Cambios Importantes"
```

**Review by SteveLauC — COMMENTED** [review]
Thanks for doing this!

Just one error I found:)

**Comment by Rikiub** [comment]
> Thanks for doing this!
> 
> Just one error I found:)

Done! Thanks to check!

**Review by SteveLauC — APPROVED** [review]

**Comment by SteveLauC** [comment]
Thanks!

**Pull request merged**